src/query/test/compatibility/test.go

```diff
@@ -371,7 +371,7 @@ func newEvalCmd(expr string, start time.Time, line int) *evalCmd {
 
 		metrics:  map[uint64]labels.Labels{},
 		expected: map[uint64]entry{},
-		m3query:  newM3QueryClient("localhost", 7201),
+		m3query:  newM3QueryClient("host.docker.internal", 7201),
```

Comment:
@kentzeng12 is this only called from a docker context? (afaict answer is yes, so this is okay).

Reply:
Since it's being run in the buildkite, which runs the test in a docker container, yes it's called in a docker context, but like you mentioned if you run this locally we run into issues.